api/specs.proto

```diff
@@ -206,14 +206,14 @@ message EndpointSpec {
 		// network. The user may then provide an alternate
 		// option for ingress load balancing which is
 		// transparent to swarm.
-		DISABLED = 1 [(gogoproto.enumvalue_customname) = "IngressRoutingDisabled"];
+		LOCAL = 1 [(gogoproto.enumvalue_customname) = "IngressRoutingDisabled"];
```

Comment:
On the swarmport that it is configured with? It could be 80/443 or it could be some other port